cygnus-common/src/main/java/com/telefonica/iot/cygnus/management/ManagementInterface.java

```diff
@@ -2137,5 +2277,27 @@ private String getFileName (String url) {
         String[] pathElements = url.split("/");
         return pathElements[pathElements.length - 1];
     } // getFileName
+    
+    /**
+     * getDescriptionsMap: return a map with basic descriptions in a instance configuration file.
+     * 
+     */
+    private Map<String,String> getDescriptionsMap () {
```

Comment:
It's true, even if the user wants to add a comment or modify the existent ones. As we commented, I'm going to create a way to read the comments before and print them after POST action